### PR TITLE
Correct validation for UU NFE Clause/Little Cup

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/rulesets.js [hunk 1]
```diff
@@ -292,6 +292,10 @@ let BattleFormats = {
 			if (!template.nfe || futureGenEvo) {
 				return [set.species + " doesn't have an evolution family."];
 			}
+			// Temporary hack for LC past-gen formats and other mashups
+			if (set.level !== 5) {
+				return [`${set.species} needs to be level 5.`];
```
Comment (Zarel) [inline data/rulesets.js]: LC requires level 5, it's not just a cap?

Comment (KrisXV) [reply data/rulesets.js]: True, fixed

File: data/rulesets.js [hunk 2]
```diff
@@ -840,8 +844,15 @@ let BattleFormats = {
 	uunfeclause: {
 		effectType: 'ValidatorRule',
 		name: 'UU NFE Clause',
-		desc: "Bans all NFE Pokemon, except Scyther, from [Gen 3] UU.",
-		// Implemented in mods/gen3/rulesets.js
+		desc: "Bans all NFE Pokemon, barring some exceptions",
```
Comment (Zarel) [inline data/rulesets.js]:
```suggestion
		desc: "Bans all NFE Pokemon, except Scyther in Gen 3",
```

File: data/rulesets.js [hunk 3]
```diff
@@ -292,6 +292,10 @@ let BattleFormats = {
 			if (!template.nfe || futureGenEvo) {
 				return [set.species + " doesn't have an evolution family."];
 			}
+			// Temporary hack for LC past-gen formats and other mashups
+			if (set.level > 5) {
+				return [`${set.species} needs to be level 5 or lower.`];
```
Comment (Zarel) [inline data/rulesets.js]:
```suggestion
				return [`${set.species} can't be above level 5 in Little Cup formats.`];
```

**Pull request merged**